src/test/java/com/wikia/webdriver/testcases/desktop/discussions/UploadingImageTests.java

```diff
@@ -79,17 +75,17 @@ public void userCanUploadImageToTheirPostOnDesktop() {
 
   @Test(groups = DESKTOP)
   @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
-  @Execute(onWikia = DESKTOP_COMMUNITY, asUser = User.USER_3)
+  @Execute(onWikia = "qadiscussions", language = "de", asUser = User.USER_3)
   public void userCanUploadImageToTheirReplyOnDesktop() {
-    PostDetailsPage page = new PostDetailsPage().open(setUp(DESKTOP_COMMUNITY).getId());
+    PostDetailsPage page = new PostDetailsPage().open(setUp("qadiscussions", "de").getId());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "qadiscussions" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)